src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +45,11 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("SELECT m from Movie m", Movie.class)
```

Comment:
it would be better to use one style in queries (or all key words with uppercase or all words without uppercase )